### PR TITLE
Fix: Unstyled Focus rectangle on error details page

### LINKED ISSUE #2587: [Bug] Unstyled focus rect shows on extension error page
The `<summary>` styles should align with those used in [`src/devtools/views/pages/results/hint.css`](https://github.com/webhintio/hint/blob/master/packages/extension-browser/src/devtools/views/pages/results/hint.css). May be worth extracting into a shared control in [`src/devtools/views/controls`](https://github.com/webhintio/hint/tree/master/packages/extension-browser/src/devtools/views/controls).

### PR DESCRIPTION
**Pull request by pratiyush2706**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] ~Added/Updated related documentation.~
- [ ] ~Added/Updated related tests.~

## Short description of the change(s)

Resolves #2587 

### Preview
Screen Shot 2019-07-11 at 8 19 18 AM

### CODE REVIEW

**Comment by antross** [comment]
@pratiyush2706 Anything left for this PR or is it ready to be moved from "Draft" to "Ready for review"?

**Comment by pratiyush2706** [comment]
> @pratiyush2706 Anything left for this PR or is it ready to be moved from "Draft" to "Ready for review"?

@antross I was just checking whether the CSS file could be made in shared context since 2 files have the same properties. What's your opinion?

**Comment by antross** [comment]
> I was just checking whether the CSS file could be made in shared context since 2 files have the same properties. What's your opinion?

Sounds reasonable. 

What do you think about adding a new `summary.tsx` component to the `controls` folder which contains the shared styles and importing that in the two places we need it? That would be similar to what we did for `controls/label.tsx`.

**Comment by pratiyush2706** [comment]
> > I was just checking whether the CSS file could be made in shared context since 2 files have the same properties. What's your opinion?
> 
> Sounds reasonable.
> 
> What do you think about adding a new `summary.tsx` component to the `controls` folder which contains the shared styles and importing that in the two places we need it? That would be similar to what we did for `controls/label.tsx`.

I was thinking something similar. So, let me make the changes and then I will make this PR ready for review.

**Comment by pratiyush2706** [comment]
Both these pages still work fine.

Screen Shot 2019-07-12 at 12 18 21 AM
Screen Shot 2019-07-12 at 12 18 17 AM

**Comment by antross** [comment]
@pratiyush2706 FYI, looks like you need to rebase and resolve some conflicts.

**Comment by pratiyush2706** [comment]
> @pratiyush2706 FYI, looks like you need to rebase and resolve some conflicts.

Done.

File: packages/extension-browser/src/devtools/views/controls/summary.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.summary {
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/controls/summary.css]: Probably should rename the class `.root` for consistency with our other controls.

File: packages/extension-browser/src/devtools/views/controls/summary.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+export type Props = ElementProps<'summary'>;
+
+/**
+ * Summary configuration inputs with common styles.
+ */
+
+const Summary = ({ className, ...props }: Props) => {
+    return (
+        <summary className={styles.summary}{...props} />
```
Comment (antross) [inline packages/extension-browser/src/devtools/views/controls/summary.tsx]: This currently will drop any `className` passed into the component. We can use the same approach as `label.tsx` to preserve it (also assuming the rename of the base `className` to `root`):
```suggestion
        <summary className={`${styles.root} ${className || ''}`} {...props} />
```

**Comment by antross** [comment]
@utsavized Can you take a look at this? Thanks!

**Pull request merged**